Tests/AnyCodableTests/AnyEncodableTests.swift

```diff
@@ -146,9 +120,38 @@ class AnyEncodableTests: XCTestCase {
             "string": "string",
             "array": "[1, 2, 3]",
         }
-        """.data(using: .utf8)!
-        let expectedJSONObject = try JSONSerialization.jsonObject(with: expected, options: []) as! NSDictionary
+        """
 
-        XCTAssertEqual(encodedJSONObject, expectedJSONObject)
+      try XCTAssertJsonAreIdentical(json, expected)
     }
 }
+
+
+
+func XCTAssertJsonAreIdentical(_ expression1: String, _ expression2: String, options: JSONSerialization.WritingOptions? = nil) throws {
```

Comment:
Helper functions to test that two serialized JSON (either `String` or `Data`) are equivalent.